src/createServer.js

```diff
@@ -1,3 +1,45 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { catchError } = require('./catchError');
+const { getUrlParams } = require('./getUrlParams');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+function createServer() {
+  return http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const [urlText, urlQuery] = req.url.split('?');
```

Comment:
Better move this `.split` inside `getUrlParams` and pass `req.url` to that function. No need to leave part of url processing outside.